docs/ps.md

```diff
@@ -372,7 +371,7 @@ DESCRIPTION
   Called with no arguments shows the current dyno size.
 
   Called with one argument sets the size.
-  Where SIZE is one of free|hobby|standard-1x|standard-2x|performance
+  Where SIZE is one of eco|hobby|standard-1x|standard-2x|performance
```

Comment:
Do we need to update hobby to basic as well?

Reply:
👍